crates/sargon/src/lib.rs

```diff
@@ -59,6 +59,8 @@ pub mod prelude {
     pub(crate) use uuid::Uuid;
 
     pub(crate) use enum_as_inner::EnumAsInner;
+    pub(crate) use native_radix_engine_toolkit::receipt::RuntimeToolkitTransactionReceipt as RetRuntimeToolkitTransactionReceipt;
```

Comment:
Ah this should be called `ScryptoRuntimeToolkitTransactionReceipt` I think... So I've been using the `RET` or `Ret` prefix for types which comes from a crate in the Radix-Engine-Toolkit Git repository. And I've been using `Scrypto` prefix for types which comes from a crate in the Scrypto Git repo, and `native_radix_engine_toolkit` crate comes from Scrypto repo...